apps/api/v2/src/modules/organizations/controllers/inputs/get-users-input.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsSingleEmailOrEmailArray } from "@/modules/organizations/controllers/inputs/validator/isEmailStringOrArray";
+import { ApiProperty } from "@nestjs/swagger";
+import { IsOptional, IsString, Validate } from "class-validator";
+
+export class GetUsersInput {
+  @IsOptional()
+  @Validate(IsSingleEmailOrEmailArray)
+  @ApiProperty({
+    description: "The email address or an array of email addresses to filter by",
+  })
+  email!: string | string[];
```

Comment:
Important - we should define inputs and outputs in the platform types package. I am refactoring v2 schedules and [here](https://github.com/calcom/cal.com/pull/15457/files#diff-c14655a575027fff668fed90cdcdd438bb385d4b8ac4db971829e0ee7368733e) is example of defining input and [here](https://github.com/calcom/cal.com/pull/15457/files#diff-fdfffb5ca46b3e58238144c37c879b8a4859bc45fdaa257d7f5fb98f2c1a1ae2) and output.

We should do it because in the future if we build atoms that display organization users, then we need to know the outputs of them, so if we define inputs and outputs in "packages/platform/types" then we can re-use them.

However, we only store types of inputs and outputs of **data** not return of api that returns also status success or error. Aka if you check your "OrganizationsController.getOrganizationUsers" function you see return:
```
Promise<ApiResponse<ListUsersResponseDto>>
```
it is wrapped with "ApiResponse" which is also defined in platform types package, so you only need to define `ListUsersResponseDto` type in platform types, so that then atoms can import "ApiResponse" and `ListUsersResponseDto` and reconstruct type of the api return.